server/views/pages/overview/partials/workAndInterestsTab/_inductionLongQuestionSet.njk

```diff
@@ -193,7 +193,14 @@ questions are different.
             </ul>
           </dd>
           <dd class="govuk-summary-list__actions govuk-!-display-none-print">
-            <a class="govuk-link" href="{{ ciagInductionUrl }}/plan/create/{{ prisonerSummary.prisonNumber }}/skills/update">
+            {%- set skillsChangeLinkUrl -%}
+              {%- if featureToggles.induction.update.skillsAndInterestsSectionEnabled -%}
```

Comment:
I'm not suggesting we change this to `personalSkillsAndInterestsSectionEnabled` - it's clear which skills or interests we're referring to when it's within the context of "skills and interests".